docs/getting-started.md

```diff
@@ -278,8 +278,7 @@ you're good to go!
 Currently, the image file downloaded from step 1 was built in 2020.
 `apt` does not work out of the box.
 
-To fix `apt` in general, `/etc/apt/sources.list` needs to be changed to match
-https://github.com/ev3dev/docker-library/blob/master/ev3dev-stretch/layers/debian/etc/apt/sources.list
+To fix `apt` in general, `/etc/apt/sources.list` needs to be changed to match [SourcesList]
```

Comment:
```suggestion
To fix `apt` in general, `/etc/apt/sources.list` needs to be changed to match [this file][SourcesList]
```

"SourcesList" is OK as a lookup key but seems a little odd as the actual text to display.